.github/workflows/main.yml

```diff
@@ -6,12 +6,12 @@ on:
   workflow_dispatch:
   pull_request:
   push:
-    branches:    
+    branches:
       # Push events on develop branch
       - develop
 
 # This workflow has the following jobs:
-# linters: Kotlin and Protobuf Linting
+# linters: Kotlin, Protobuf and Java Linting
```

Comment:
Change to `# linters: Java, Kotlin, Protobuf`
(Keep them alphabetical)

Reply:
done

Reply:
resolving on behalf of Vinita